.github/workflows/build-test.yml

```
@@ -4,9 +4,11 @@ on:
   push:
     branches:
       - main
+      - depromeet/layer:main
   pull_request:
```

참조 : https://blog.outsider.ne.kr/1541

```suggestion
  pull_request_target:
```

---

와 빠르다 !! 적용해볼게요

---

🚀

---

흠,, 보니까 애초에 포크한 PR은 신뢰할 수 없다고 판단해서 이 이벤트로도 빌드는 하지 말라고 하는 것 같네요..?! ([Warning 참고](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target))